### PR TITLE
[AMORO-2701] Add maillist info in the community page

### PR DESCRIPTION
**Pull request by klion26**
Add mail list info in the community page

cc @zhoujinsong

### CODE REVIEW

**Comment by klion26** [comment]
image

**Comment by zhoujinsong** [comment]
@klion26 There is some conflicts you may want to fix.

**Comment by klion26** [comment]
@zhoujinsong thanks for the view, has been rebased, please have another look when you're free.

File: amoro-site/content/community/join-community.md [hunk 1]
```diff
@@ -10,6 +10,19 @@ Anyone interested in the Amoro project can join the community and contribute to
...
+Amoro has three mailing lists:
+- **Developers**: <dev@amoro.apache.org>  -- used for community discussions
+  - [Subscribe](mailto:dev-subscribe@amoro.apache.org)
+  - [Unsubscribe](mailto:dev-unsubscribe@amoro.apache.org)
+  - [Archive](https://lists.apache.org/list.html?dev@amoro.apache.org)
+- **Commits**: <commits@amoro.apache.org> -- distributes commit notifications
+  - [Subscribe](mailto:commits-subscribe@amoro.apache.org)
+  - [Unsubscribe](mailto:commits-unsubscribe@amoro.apache.org)
+  - [Archive](https://lists.apache.org/list.html?commits@amoro.apache.org)
+- **Private**: <private@amoro.apache.org> -- private list for the PMC to discuss sensitive issues related to the health of the project
```
Comment (zhoujinsong) [inline amoro-site/content/community/join-community.md:23]:
```suggestion
- **Private**: <private@amoro.apache.org> -- private list for the PPMC to discuss sensitive issues related to the health of the project
```

**Comment by klion26** [comment]
@zhoujinsong thanks for the review and merging!

**Pull request merged**